k8s-artifacts/api-portal/wso2apim-service.yaml

```diff
@@ -39,4 +39,16 @@ spec:
     - name: servlet-https
       port: 9443
       targetPort: 9443
-      nodePort: 32001
\ No newline at end of file
+      nodePort: 32001
+    - name: jms-port
+      port: 5672
+      targetPort: 5672
+      nodePort: 32005
+    - name: binary-receiver
+      port: 9611
+      targetPort: 9611
+      nodePort: 32006
+    - name: binary-auth
+      port: 9711
+      targetPort: 9711
+      nodePort: 32007
```

Comment:
shall we have a new line here

Reply:
Fixed with https://github.com/wso2/k8s-api-operator/pull/534